src/State/AuthFlow/reducers.js

```diff
@@ -0,0 +1,29 @@
+import initialState from '../../Reducers/initialState';
+import * as types from './types';
+
+const authReducer = (state = initialState.currentUser, action) => {
+  switch (action.type) {
+    case types.AUTHENTICATE_FETCH: {
+      return { ...state };
+    }
+    case types.AUTHENTICATE_SUCCESS: {
+      return {
+        ...state,
+        AccessKeyId: action.payload.AccessKeyId,
```

Comment:
If you want, we could spread out of `action.payload`, although that might sacrifice clarity haha